repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -26,8 +27,8 @@
     </div>
     <div id="nominal-date">
         <h4>Nominal publication date:</h4>
-        <#if document.nominalDate??>
-            <@fmt.formatDate value=document.nominalDate.time type="Date" pattern=dateFormat />
+        <#if document.nominalPublicationDate??>
```

Comment:
the same needs to be applied to `repository-data/webfiles/src/main/resources/site/freemarker/common/searchresults.ftl`

Reply:
Good spot! Now fixed.